test/unit/plugin_logging_test.rb

```diff
@@ -0,0 +1,46 @@
+require 'test_helper'
+
+class PluginLoggingTest < ActiveSupport::TestCase
+  def setup
+    @plugin_config = {:loggers => {:foreman_test_plugin => {:enabled => true}}}
+    @logging = Foreman::Plugin::Logging.new("foreman_test_plugin")
+    @logging.configure(@plugin_config)
+  end
+
+  def assert_logger_exists(name)
+    assert ::Logging::Repository.instance.has_logger?(name)
+  rescue Minitest::Assertion
+    raise Minitest::Assertion, "Failed assertion, logger #{name} does not exist"
```

Comment:
I don't think I've seen this re-raising pattern with assertions before - can't you just add the message as the second parameter to `assert` above?